README.md

```diff
@@ -57,41 +60,103 @@ If you are interested in becoming a member of the team check out the **[Pixelogi
+We have found some awesome workflows and tools to get us up and running with Golang. This repo includes a [`.vscode/settings.json`](.vscode/settings.json) & [`.vscode/extensions.json`](.vscode/extensions.json) which will allow you to automatically download the recommended extensions and keeps your repo styling in sync with all the contributors. If you find more extensions please join the [PixelogicDev Discord](https://discordapp.com/invite/8NFtvp5) and share with us so we can add it to the project!
 
 ### Grüvee Backend File Path
 
 Make sure `gruveebackend` is in your GOPATH (This helps a lot. We promise.)
 
-1.  Clone repo from Github
-2.  Open up `GOPATH/src/github.com/`
-3.  Create folder called `pixelogicdev`
-4.  `cd pixelogicdev` and move/clone `gruveebackend` into this folder
+1. Clone repo from Github
+1. Open up `GOPATH/src/github.com/`
+1. Create folder called `pixelogicdev`
+1. `cd pixelogicdev` and move/clone `gruveebackend` into this folder
 
 ### Running Functions Locally
 
 When you want to run all the functions locally, all you need to do is run `scripts/gorun.sh`. This will work out of the box on any macOS/Linux/Ubuntu system. If you are running on Windows 10 you will need to follow a [guide like this](https://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/) to get the bash system running on your machine.
 
 This script does the following:
 
-```
+```text
 Adds all the replace lines in all the go mod files
 Builds and runs the `main.go` file in the root
 ```
 
+## Auto Tagging and Deploy with Github Actions
+
+GitHub [Actions](https://docs.github.com/en/actions) is used for tagging functions when updated and pushing them to gcp. This is an automated process which has the following requirements:
+
+- Any change made in a function's folder structure (e.g. `/cmd/appleauth/**`) must be accompanied with a new version (maintained in `.version` file).
+  - If a change is made to a function and the same tag is used again, the deploy will fail!
+- All code destined for `master` should/must go through a pull request (pr). No pushes should go directly to master, let's keep good habits!
+- GitHub Actions yaml files are kept at `.github/workflows/`
+- There is **one** Actions yaml file per **function** per **trigger**. This allows multiple functions to be updated in a push and each will be tagged and deployed as needed.
+  - There is currently only a 'push to master' trigger file. Other useful triggers are on PR, which can run linting,tests, and even pushed to staging, vetting the code being staged for production deployment.
```

Comment:
[nit]

Please add space after the comma `linting,tests,`

Reply:
Fixed in latest commit